README.md

```diff
@@ -556,16 +573,17 @@ metricExporter.Start();
 4. See [sample][ai-sample] for example use.
 
 ``` csharp
-using (var tracerFactory = TracerFactory.Create(builder => builder
-    .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key")))
-{
-    var tracer = tracerFactory.GetTracer("application-insights-test");
-    var span = tracer
-        .SpanBuilder("incoming request")
-        .StartSpan();
+using var tracerFactory = TracerFactory.Create(builder => builder
+    .SetResource(Resources.CreateServiceResource("my-service"))
+    .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key"));
+var tracer = tracerFactory.GetTracer("application-insights-test");
 
-    await Task.Delay(1000);
-    span.End();
+using (DistributedContext.SetCurrent(dc))
+using (tracer.StartActiveSpan("incoming request", out var span))
+{
+    span.AddEvent("Start processing video.");
+    Thread.Sleep(TimeSpan.FromMilliseconds(10));
+    span.AddEvent("Finished processing video.");

```

Comment:
This sample does not compile. Please give me a few minutes, I'm working on an alternative code sample.

Reply:
@eddynaka Please use this instead:

```
static void Main(string[] args)
{
    DistributedContext.Carrier = AsyncLocalDistributedContextCarrier.Instance; // Enable asynclocal carrier for the context
    DistributedContext dc = DistributedContextBuilder.CreateContext(key: "my.org/keys/frontend", value: "mobile-ios9.3.5");

    using var tracerFactory = TracerFactory.Create(builder => builder
        .SetResource(Resources.CreateServiceResource("my-cloud-role-name"))
        .UseApplicationInsights(config => config.InstrumentationKey = "559ab47f-be09-4558-a0be-3ae747845bca"));

    var tracer = tracerFactory.GetTracer("application-insights-test");

    using (DistributedContext.SetCurrent(dc))
    using (tracer.StartActiveSpan("incoming request", out var span))
    {
        span.AddEvent("Start processing video."); // AI Trace message
        Thread.Sleep(TimeSpan.FromMilliseconds(10));
        span.AddEvent("Finished processing video."); // AI Trace message
    }

    // Note that a manual flush is not necessary. The Application Insights TelemetryClient will be flushed when the tracerFactory is disposed.
}
```

Reply:
@TimothyMothra , I will update the sample to include the FrontendKey (which is your "my.org/keys/frontend" string). With that, besides the key and the strings that you added, the example is the same.